### ISSUE TITLE
openapi.json adaptations for a better API doc and stubs

### ISSUE THREAD

**Issue by grololo06** [ecotaxa/ecotaxa_dev#614]
Found by @laurent-n 

It looks like adding:
```
  "servers": [{
  "url": "/api",
  "description": "Local /api"
   }]  ,
```

Straight into the openapi.json, makes the /docs directly usable for the calls (so far it does not reference /api prefix)

Also setting attributes to nullable e.g.
```

"organisation":{
  "title":"Organisation",
  "type":"string",
  "nullable": true
},

```
Allows to use openapi-generator 5.X

**Comment by grololo06** [comment]
Minimally adding into the generated doc a warning about the missing /api prefix would help new users.

**Comment by grololo06** [comment]
Fixed in 2.5.8.
